unpackaged/test/primary_affiliations/pre/classes/STG_UnmanagedInstall.cls

```diff
@@ -0,0 +1,13 @@
+public without sharing class STG_UnmanagedInstall {
+
+    //Include Affiliation mappings for test record types
+    public static void mapAccountRecordTypesForAffiliationTesting(){
+        String namespace = UTIL_Namespace.getNamespace();
+        String prefix = (String.isNotBlank(namespace)) ? namespace + '__' : '';
+
+        List<Affl_Mappings__c> mappings = new List<Affl_Mappings__c>();
+        mappings.add(new Affl_Mappings__c(Name = 'An Active Record Type', Account_Record_Type__c = 'An_Active_Record_Type', Primary_Affl_Field__c = prefix + 'Primary_Active_Account__c'));
+        mappings.add(new Affl_Mappings__c(Name = 'An Inactive Record Type', Account_Record_Type__c = 'An_Inactive_Record_Type', Primary_Affl_Field__c = prefix + 'Primary_Inactive_Account__c'));
```

Comment:
We do not need the namespacing here since all of these fields should be unmanaged and treated as though they were customer configurations.